tests/cypress/support/grafana_selectors.ts

```diff
@@ -0,0 +1,28 @@
+const grafanaVersion = 10;
```

Comment:
have you looked into [@grafana/e2e-selectors](https://www.npmjs.com/package/@grafana/e2e-selectors) if this could help us in the long run?
i would be okay with updating this dependency to 10

Reply:
I tried using @grafana/e2e-selectors but they were not working with Cypress tests, so I did a workaround. I need to explore it a little more